Assets/MixedRealityToolkit/UX/Scripts/Tooltip/ToolTipBackgroundBlob.cs

```diff
@@ -0,0 +1,200 @@
+            Vector3 localContentSize = toolTip.LocalContentSize;
+            Vector3 localContentOffset = toolTip.LocalContentOffset;
+
+            // Get the size of the mesh and use this to adjust the local content size on the x / y axis
+            // This will accomodate meshes that aren't built to 1,1 scale
+            Bounds meshBounds = BackgroundRenderer.GetComponent<MeshFilter>().sharedMesh.bounds;
+            localContentSize.x /= meshBounds.size.x;
+            localContentSize.y /= meshBounds.size.y;
+            localContentSize.z = 1;
+
+            localContentBounds.size = Vector3.one; //localContentSize;
+            localContentBounds.center = localContentOffset;
+        }
+
+        private void Update ()
+        {
+            // Adjust center and size by velocity
+            Vector3 currentPosition = PositionTarget.position;
+            velocity = Vector3.Lerp(velocity, lastPosition - currentPosition, 1f / blobInertia * Time.deltaTime);
+            Vector3 currentDistortion = -velocity * blobDistortion;
+            distortion = Vector3.Lerp(distortion, currentDistortion, 1f / blobDistortion * Time.deltaTime);
+
+            inertialContentBounds.center = inertialContentBounds.center + velocity;
+            Vector3 size = inertialContentBounds.size + distortion;
+            inertialContentBounds.size = size;
+
+            Vector3 currentRotation = Vector3.zero;
+            currentRotation.x = velocity.x * 360;
+            currentRotation.z = velocity.z * 360;
+            currentRotation.y = velocity.y * 360;
+            currentRotation = RotationTarget.TransformDirection(currentRotation);
+            rotation = Vector3.Lerp (rotation, currentRotation, 1f / blobRotation * Time.deltaTime);
+
+            // Correct the center and size
+            inertialContentBounds.center = Vector3.Lerp(inertialContentBounds.center, localContentBounds.center, Time.deltaTime * positionCorrectionStrength);
+            inertialContentBounds.size = Vector3.Lerp(inertialContentBounds.size, localContentBounds.size, Time.deltaTime * distortionCorrectionStrength);
+            RotationTarget.localRotation = Quaternion.Lerp(PositionTarget.localRotation, Quaternion.identity, Time.deltaTime * rotationCorrectionStrength);
+
+            // Apply the center and size
+            PositionTarget.localPosition = inertialContentBounds.center + blobOffset;
+            DistortionTarget.localScale = inertialContentBounds.size;
+            RotationTarget.Rotate(velocity * blobRotation * 360);
+
+            // Adjust the tool tip attach position
+            toolTip.AttachPointPosition = AttachPointOffset.position;
+
+            lastPosition = currentPosition;
+            
+
+        }
+
+        private Vector3 lastPosition;
+        private Vector3 velocity;
+        private Vector3 distortion;
+        private Vector3 rotation;
+        private Bounds localContentBounds;
+        private Bounds inertialContentBounds;
```

Comment:
move these to the other private fields declared at the top of the class.

Reply:
fields moved.